hack/e2e/run.sh

```diff
@@ -231,15 +236,16 @@ else
 
     set -x
     set +e
+    #TODO: revert #246 once a stable release including windows nodeSelector change is available; set to v1.27.0-alpha.3 for testing
     kubetest2 noop \
       --run-id="e2e-kubernetes" \
       --test=ginkgo \
       -- \
       --skip-regex="${GINKGO_SKIP}" \
       --focus-regex="${GINKGO_FOCUS}" \
-      --test-package-version=$(curl https://storage.googleapis.com/kubernetes-release/release/stable-$packageVersion.txt) \
-      --parallel=25 \
```

Comment:
After #1526 is merged is is safe to kill `$GINKGO_PARALLEL` entirely? Or do we still need to run this test slower?

Reply:
Discussed offline and decided to keep `$GINKGO_PARALLEL` over provisioning more nodes; 3 nodes is not sufficient for running the tests with `--parallel=25` as testing pods are unable to be scheduled in the 5 minute window, `tolerationSeconds` is set for 300s (5 minutes).